### DISCUSSION TITLE
Adobe Commerce Live Search Integration

### DISCUSSION THREAD

**Discussion by sandipansaha** [breezefront/community#58] [2. Ideas]
Hi Support,

Adobe is now recommending Live Search with its commerce and cloud projects. We love to build stores with the breeze and built a few cloud projects with it.
Is there any plan of having integration compatibility for the Live search feature?

https://business.adobe.com/in/products/magento/live-search.html
https://marketplace.magento.com/magento-live-search.html
https://experienceleague.adobe.com/docs/commerce-merchant-services/live-search/onboard/install.html?lang=en

Thanks

**Comment by vovayatsyuk** [comment]
Unfortunately, we do not have access to Live search and we do not have customers that use this feature.
